mobile/library/common/engine_common.cc

```diff
@@ -101,7 +101,10 @@ EngineCommon::EngineCommon(std::shared_ptr<Envoy::OptionsImplBase> options) : op
   base_ = std::make_unique<StrippedMainBase>(
       *options_, real_time_system_, default_listener_hooks_, prod_component_factory_,
       std::make_unique<PlatformImpl>(), std::make_unique<Random::RandomGeneratorImpl>(), nullptr,
-      create_instance);
+      create_instance,
+      // `set_new_handler` is false because the application using Envoy Mobile should decide how to
+      // handle `new` memory allocation failures.
+      /*set_new_handler=*/false);
```

Comment:
nit: Consider doing:
```
  // `set_new_handler` is false because the application using Envoy Mobile should decide how to
  // handle `new` memory allocation failures.
  const bool set_new_handler = false;
  base_ = std::make_unique<StrippedMainBase>(
      *options_, real_time_system_, default_listener_hooks_, prod_component_factory_,
      std::make_unique<PlatformImpl>(), std::make_unique<Random::RandomGeneratorImpl>(), nullptr,
      create_instance, set_new_handler);
```

This moves the comment outside of the function invocation and uses a named local to avoid the need for the `/*=*/` comment.

Reply:
done